rapids-cmake/test/add.cmake

```diff
@@ -89,6 +89,15 @@ function(rapids_test_add)
   if(TARGET ${command_or_target})
     set(command "$<TARGET_FILE:${command}>")
   endif()
+  if(DEFINED _RAPIDS_TEST_INSTALL_TARGET)
+    if(NOT TARGET ${_RAPIDS_TEST_INSTALL_TARGET})
+      message(FATAL_ERROR "rapids_add_test given INSTALL_TARGET \"${_RAPIDS_TEST_INSTALL_TARGET}\", which does not exist"
+      )
+    endif()
+    set(target_to_install ${_RAPIDS_TEST_INSTALL_TARGET})
+  else()
+    set(target_to_install ${command_or_target})
```

Comment:
Can we add a check that `target_to_install` is a valid target here, just to be safe and provide users more useful diagnostics?
```suggestion
    set(target_to_install ${command_or_target})
    if (NOT TARGET ${target_to_install})
      message(FATAL_ERROR "rapids_add_test could not infer a target to install")
    endif()
```

Reply:
This would regress the use case of running a command that does not involve a target at all. Is that something we need?

Reply:
We could make it a warning instead.

Reply:
Good point. Since that is a valid use case, perhaps `VERBOSE` is the appropriate mode.

Reply:
OK, I've made it a `VERBOSE` message.